### DISCUSSION TITLE
Vitesse + '@cheprasov/qrcode' import problem

### DISCUSSION THREAD

**Discussion by Andylife333** [antfu/vitesse#274] [Q&A] [answered]
In dev mode

`import { QRCodeSVG } from '@cheprasov/qrcode'`

runing correctly. But in build mode systax error 

`Named export 'QRCodeSVG' not found .... using:`

`import pkg from '@cheprasov/qrcode';
const { QRCodeSVG } = pkg;
`

Builder say good, but in dev mode the construction does not work.

Switch the code between dev and build mode all the time is a bad idea.
How to fix it?

**Comment by spokospace** [comment]
It seems to me there is issue with some imports after upgrade vite-ssg, I have similar problem with scss files.

**Comment by Andylife333** [comment, marked as answer]
Solution find!

`import * as pkg from '@cheprasov/qrcode';
const { QRCodeSVG } = pkg;`

Work in dev and in build mode.
